modules/core/src/shaderlib/project/project.glsl.js

```diff
@@ -115,13 +124,22 @@ vec4 project_position(vec4 position) {
   return project_offset_(position_modelspace);
 }
 
+vec4 project_position(vec4 position) {
+  return project_position(position, vec2(0.0, 0.0));
```

Comment:
Idea: maybe defined a constant `const vec2 zero_64(0.0, 0.0)` and use in all these places to make code more self documenting?

Reply:
Done